### PR TITLE
feature/557

### LINKED ISSUE #557: When creating/instantiating 3D model (before running it), screen comment is "Simulating..."
Screenshot 2022-06-10 at 09 01 54

### PR DESCRIPTION
**Pull request by aranega**
Fixes #557, when an 3D models are created/instantiated, the spinner message always displays:

"Simulating..."

The message now is dependent on the action that triggers the spinner.

### CODE REVIEW

File: webapp/redux/middleware/middleware.js [hunk 1]
```diff
@@ -342,14 +342,14 @@ export default (store) => (next) => (action) => {
     case CREATE_NETWORK: {
       next(GeppettoActions.waitData('Instantiating the NetPyNE Model', GeppettoActions.layoutActions.SET_WIDGETS));
```
Comment (ddelpiano) [inline webapp/redux/middleware/middleware.js:343]: @aranega this looks redundant (unless I am missing something) with the implementation you did, you will have this triggered first and then another waitData later with the wait message you implemented.

File: webapp/redux/middleware/middleware.js [hunk 2]
```diff
@@ -342,14 +342,14 @@ export default (store) => (next) => (action) => {
...
 
-      checkParametersThen(() => instantiateNetwork({}))
+      checkParametersThen(() => instantiateNetwork({}), false, "Creating the NetPyNE Model")
```
Comment (ddelpiano) [inline webapp/redux/middleware/middleware.js:345]: I would use the same message we had in the line above -> "Instantiating the NetPyNE Model"

File: webapp/redux/middleware/middleware.js [hunk 3]
```diff
@@ -342,14 +342,14 @@ export default (store) => (next) => (action) => {
...
 
 
       break;
     }
     case CREATE_SIMULATE_NETWORK: {
 
-      checkParametersThen(() => simulateNetwork({ allTrials: false }))
+      checkParametersThen(() => simulateNetwork({ allTrials: false }), false, "Creating and simulating the NetPyNE Model")
```
Comment (ddelpiano) [inline webapp/redux/middleware/middleware.js:352]: replace Creating with Instantiating here as well

**Review by ddelpiano — CHANGES_REQUESTED** [review]

File: webapp/redux/middleware/middleware.js (hunk 1 above)
Comment (aranega) [reply webapp/redux/middleware/middleware.js:343]: @ddelpiano Thanks for the review! I wasn't sure I could remove it safely (that there was no side-effect somewhere), so I didn't want to disrupt that much the original code. I'll remove L343 in favour of the message display that is inside the `checkParametersThen` method.

**Review by ddelpiano — APPROVED** [review]

**Comment by ddelpiano** [comment]
Looks good to me, also the functional testing looks fine.
Thanks @aranega !

**Comment by ddelpiano** [comment]
@enicolasgomez please review this as well and when good for you merge it. Thanks!

**Pull request merged**